llama-index-networks/llama_index/networks/contributor/query_engine/client.py

```diff
@@ -10,7 +10,7 @@
 import aiohttp
 
 
-class ContributorClientSettings(BaseSettings):
```

Comment:
Is this a breaking change? Can we alias the old names maybe?

Reply:
Yeah, you know what I aliased it in one of the __init__ files, but I think you're right I should do it here, and merely import it in the __init__ files.

Reply:
done!